src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/hooks.js

```diff
@@ -210,7 +210,9 @@ export const typeRowHooks = ({
     if (typeKey === ProblemTypeKeys.DROPDOWN) {
       if (correctAnswerCount > 1) {
         answers.forEach(answer => {
-          updateAnswer({ ...answer, correct: false });
+          if (answer.correct) {
+            updateAnswer({ ...answer, correct: false });
+          }
```

Comment:
I'm confused. I have seen that this fixes the issue, but I haven't the faintest idea why?